zone/command.cpp

```diff
@@ -239,7 +239,7 @@ int command_init(void)
 		command_add("nukeitem", "[Item ID] - Removes the specified Item ID from you or your player target's inventory", AccountStatus::GMLeadAdmin, command_nukeitem) ||
 		command_add("object", "List|Add|Edit|Move|Rotate|Copy|Save|Undo|Delete - Manipulate static and tradeskill objects within the zone", AccountStatus::GMAdmin, command_object) ||
 		command_add("oocmute", "[0|1] - Enable or Disable Server OOC", AccountStatus::GMMgmt, command_oocmute) ||
-		command_add("opcode", "opcode management", AccountStatus::GMImpossible, command_opcode) ||
+		command_add("opcode", "- Reloads all server patches", AccountStatus::GMImpossible, command_opcode) ||
```

Comment:
Probably can remove the dash at the beginning. Almost no current descriptions start with a dash